plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -45,53 +56,69 @@ def __init__(
         hazard: str,
         folder_name: str,
         together_api_key: TogetherApiKey,
-        benign_prompts: bool = False,
+        persona_types: List[str],
     ):
         super().__init__(uid)
         assert hazard in HAZARDS_MAPPING, f"Invalid hazard type {hazard}."
+        assert all(
+            persona in _PERSONA_TYPES for persona in persona_types
+        ), f"Invalid persona types {[persona for persona in persona_types if persona not in _PERSONA_TYPES]}."
```

Comment:
Minor: I think this is slightly cleaner if you make the list of `invalid_personas` then assert the list is empty.

Alternatively, I think if Persona is an enum we don't need validation.

Reply:
Done